src/__tests__/bulkUploads.int.test.ts

```diff
@@ -1,213 +1,205 @@
 import request from 'supertest';
 import { app } from '../app';
-import {
-  db,
-  loadTableMetrics,
-} from '../database';
+import { db, loadTableMetrics } from '../database';
 import { expectTimestamp } from '../test/utils';
 import { mockJwt as authHeader } from '../test/mockJwt';
 import { BulkUploadStatus } from '../types';
 
 const agent = request.agent(app);
 
 describe('GET /bulkUploads', () => {
-  it('returns an empty Bundle when no data is present', async () => {
-    await agent
-      .get('/bulkUploads')
-      .set(authHeader)
-      .expect(200, {
-        total: 0,
-        entries: [],
-      });
-  });
+	it('returns an empty Bundle when no data is present', async () => {
+		await agent.get('/bulkUploads').set(authHeader).expect(200, {
```

Comment:
bah -- see this is an example of me not loving the wrapping defaults of prettier (womp womp)

Reply:
Yeah, same. :disappointed:

Reply:
Discussion here: https://github.com/prettier/prettier/issues/7884